src/gui/optionsdlg.cpp

```diff
@@ -259,7 +259,7 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     connect(m_ui->autoRunBox, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->autoRun_txt, &FileSystemPathEdit::selectedPathChanged, this, &ThisType::enableApplyButton);
 
-    const QString autoRunStr = QString::fromUtf8("%1\n    %2\n    %3\n    %4\n    %5\n    %6\n    %7\n    %8\n    %9\n    %10\n%11")
+    const QString autoRunStr = QString("%1\n    %2\n    %3\n    %4\n    %5\n    %6\n    %7\n    %8\n    %9\n    %10\n%11")
```

Comment:
Why not `QLatin1String`?

Reply:
QLatin1String doesn't have arg() method.

Reply:
QString::fromLatin1 would be better for latin1 string initialization. It also fit here.

Reply:
>QString::fromLatin1 would be better for latin1 string initialization.  It also fit here.

Is it really necessary? I mean all other similar instances uses QString(), now all of a sudden QString::fromLatin1 is preferred, isn't it a bit overzealous for such non-performance critical code?

Reply:
@evsh constantly focuses on that. I think it is unnecessary in many cases (especially use of QLatin1Char).